blockstack/lib/fast_sync.py

```diff
@@ -597,11 +597,19 @@ def logerr(s):
         logerr("No such directory {}".format(working_dir))
         return False
 
-    # go get it 
-    import_path = fast_sync_fetch(working_dir, import_url)
+    import_path = None
```

Comment:
Please don't do this.  If you have to download the snapshot file, you should do so separately.  This method shouldn't fail because the computer is offline.

Reply:
This change allows the method to _work_ when offline and passed a file. Previously it required a url and always attempted download.

Reply:
Oh, my bad.  Then this is definitely an improvement :+1: